### PR TITLE
Add alternative to install [louvain] extra requires

### LINKED ISSUE #451: `pip install cellxgene[louvain]` doesn't work with zsh
This seems to be more of a "me" problem because my shell is wacky (oh-my-zsh) but the instructions to do `pip install cellxgene[louvain]` didn't work for me:

```
(cellxgene) 
 Thu 15 Nov - 12:41  ~/Downloads 
  pip install cellxgene[louvain]
zsh: no matches found: cellxgene[louvain]
```

But if I switch to the `bash` shell it works just fine:

```
(cellxgene) bash             
bash-3.2$ pip install cellxgene[louvain]
Requirement already satisfied: cellxgene[louvain] in /anaconda3/envs/cellxgene/lib/python3.6/site-packages (0.0.2)
Requirement already satisfied: anndata>=0.6.12 in /anaconda3/envs/cellxgene/lib/python3.6/site-packages (from cellxgene[louvain]) (0.6.13)
Requirement already satisfied: click>=6.7 in /anaconda3/envs/cellxgene/lib/python3.6/site-packages (from cellxgene[louvain]) (7.0)
Requirement already satisfied: pandas>=0.23.1 in /anaconda3/envs/cellxgene/lib/python3.6/site-packages (from cellxgene[louvain]) (0.23.4)
.... more output ....
```

### LINKED ISSUE #464: Document system requirements for `cellxgene[prepare]`
I think `pip install cellxgene[louvain]` isn't going to work for a lot of people. On ubuntu you need

```bash
apt-get install build-essential libxml2-dev python3-dev zlib1g-dev
```
and on centos
```bash
yum install gcc gcc-c++ libxml2-devel make python36u-devel zlib-devel
```
(or however you install python headers)

### PR DESCRIPTION
**Pull request by csweaver**
Updated the install instructions to describe how to install the requirements to perform louvain clustering individually. Using setup.py's extra requires method `pip install scanpy[louvain]` does not work on all environments.

@davidrecordon please ensure that this passes license requirements
@bkmartinjr / @freeman-lab please check that the instructions are clear and in the right place

Fixes #451

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,12 @@ cellxgene prepare --help
 pip install cellxgene[louvain]
 ```
 
+On some environments installing the extra requires for lovain as above does not work. Alternatively, you can install these dependencies by hand in addition to the cellxgene package.
```
Comment (bkmartinjr) [inline README.md]: Suggest rewording:   

If the aforementioned optional package installation fails, you can also install these packages directly.  For example, `pip install louvain python-igraph`.

**Comment by mckinsel** [comment]
I think this fixes #451 and not #464 . That new pip command will still fail if you're missing the system dependencies.

File: README.md (hunk 1 above)
Comment (neuromusic) [reply README.md]: +1

Comment (csweaver) [reply README.md]: 👍

**Comment by freeman-lab** [comment]
maybe to address @mckinsel 's issue you should add another line saying "and on Ubuntu you'll need to first run... and on CentOS you'll need to run... "?

has anyone run the extra installs on a "fresh" Mac OSX machine? i don't recall having issues...

**Comment by csweaver** [comment]
@freeman-lab I'll address that in another PR. I've got some insights now that I've built on a docker container.

**Comment by csweaver** [comment]
Got approval from David Recordon on wording -- merging!
